### PR TITLE
umount /mnt prior to exporting zpool

### PR DESCRIPTION
**Pull request by Mic92**
potentially fixes https://github.com/nix-community/disko/issues/434

### CODE REVIEW

File: src/nixos-anywhere.sh [hunk 1]
```diff
@@ -476,7 +476,9 @@ if [[ ${copy_host_keys-n} == "y" ]]; then
 fi
 nixos-install --no-root-passwd --no-channel-copy --system "$nixos_system"
 if command -v zpool >/dev/null; then
-  zpool export -a || : # we always want to export the zfs pools so people can boot from it without force import
+  # we always want to export the zfs pools so people can boot from it without force import
+  umount -Rv /mnt/
+  zpool export -a || :
```
Comment (phaer) [inline src/nixos-anywhere.sh:481]:
```suggestion
  zpool export -a || true
```

I know it's unchanged, but i find this colon syntax confusing every time i see it :D

**Comment by phaer** [comment]
@mergifyio refresh

**Comment by phaer** [comment]
Ah https://buildbot.thalheim.io/ was restarted at the wrong moment. I'll accept the risk, hope that's okay

**Comment by Mic92** [comment]
I was rolling out https://github.com/Mic92/buildbot-nix/pull/16

**Pull request merged**